README.md

```diff
@@ -308,6 +308,127 @@ for vehicle in vehicle_path_list:
 
 All of the simulations will be launched in one go. The user can check the status of the simulations and retrieve the results when they are ready. Check the FAQ section for more information on how to do this.
 
+## Task management
+
+
+### Using the `Task` class
+
+As you may have noticed, both the sync and async runs of the scenarios show above return a `Task` object to the user.
+This object provides several methods to manage a task submitted to the **Inductiva API**.
+For instance, you can get information about the task (its current status, when it started, information on the machine it is running, ...), kill it while
```

Comment:
a specific task

Reply:
I think instead of using "(" and giving examples, you should list all the things we can do.